migrations/1576570802025-removeUnnecessaryLocalSystems.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module 'mongoose'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4689006)